cmd/ui/src/views/QA/QAV2.tsx

```diff
@@ -0,0 +1,166 @@
+// Copyright 2023 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { Alert, AlertTitle, Box, Grid, Link, Typography } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
+import {
+    ActiveDirectoryPlatformInfo,
+    AzurePlatformInfo,
+    DataSelector,
+    DomainInfo,
+    LoadingOverlay,
+    PageWithTitle,
+    SelectedDomain,
+    TenantInfo,
+    useEnvironment,
+} from 'bh-shared-ui';
+import { useEffect, useState } from 'react';
+import { dataCollectionMessage } from './utils';
+
+const useStyles = makeStyles((theme) => ({
+    container: {
+        '& div:first-child': {
+            backgroundColor: theme.palette.neutral.tertiary,
+        },
+    },
+}));
+
+const QualityAssuranceV2: React.FC = () => {
```

Comment:
I made a v2 for the QA component to consolidate the state in a similar way as `GroupManagementContent` because they behave similarly from a selected environment perspective.